saleor/tests/e2e/orders/test_apply_better_promotion_to_product.py

```diff
@@ -0,0 +1,204 @@
+import pytest
+
+from .. import DEFAULT_ADDRESS
+from ..channel.utils import create_channel
+from ..product.utils import (
+    create_category,
+    create_product,
+    create_product_channel_listing,
+    create_product_type,
+    create_product_variant,
+    create_product_variant_channel_listing,
+    get_product,
+)
+from ..promotions.utils import create_promotion, create_promotion_rule
+from ..utils import assign_permissions
+from .utils import draft_order_create, order_lines_create
+
+
+def prepare_product(
+    e2e_staff_api_client,
+    permission_manage_products,
+    permission_manage_channels,
+    permission_manage_product_types_and_attributes,
+    permission_manage_discounts,
+    permission_manage_orders,
+    channel_slug,
+    variant_price,
+    first_promotion_name,
+    first_discount_value,
+    first_discount_type,
+    first_rule_name,
+    second_promotion_name,
+    second_discount_value,
+    second_discount_type,
+    second_rule_name,
+):
```

Comment:
It's getting worse. IMO we should refactor this on the main branch, rebase and create the promotions in separate methods.

Reply:
PR with refactoring is in progress https://github.com/saleor/saleor/pull/13693. I will wait until it will be done and then update PR :)

Reply:
@IKarbowiak I think it is better to merge this test now than waiting for my changes - I will be refactoring all e2e tests anyway once the final shape of the helper will be agreed on.